test/unit/Chainweb/Test/Pact5/TransactionExecTest.hs

```diff
@@ -896,3 +791,5 @@ testLogger :: IO GenericLogger
 testLogger = do
     logLevel <- liftIO getTestLogLevel
     pure $ genericLogger logLevel T.putStrLn
+
+-- TODO: explicit test that signing for no caps is equivalent to signing for GAS
```

Comment:
still valid?

Reply:
Still valid, yes. I want to handle this in this PR.

Reply:
Done.